stories/configMock.js

```diff
@@ -42,10 +42,8 @@ export default {
         defaultCasUrl: "https://olson.cyverse.org/cas",
         defaultCasValidate: "validate",
     },
-    metadata: {
-        communityAttr: "cyverse-community",
-        instantLaunchLocationAttr: "ui_location",
-        instantLaunchDashboard: "dashboard",
-        instantLaunchNavDrawer: "nav_drawer",
```

Comment:
Hopefully removing these from the configMock is fine? doesn't seem like they moved somewhere else

Reply:
Yep, I happened to notice that the actual configs were removed in a previous PR, so I thought I'd clean these up while I was making changes here.